docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1040,6 +1041,29 @@
           on appropriate nodes.</para>
         </sect4>
       </sect3>
+
+      <sect3 id="ENV_values_yaml">
+        <title>Environment Values</title>
+
+        <para>You can define Environment Variables in a YAML file. The
+        environment values are defined under the
+        <emphasis>global.env:</emphasis> portion of the delivered values.yaml
+        as illustrated below.</para>
+
+        <programlisting>global:
+  env:
+  - name: SMTPserver 
+  - value: mysmtpserver</programlisting>
```

Comment:
There shouldn't be a leading '-' before value, otherwise it will create a new list entry, and fail the schema validation.